src/libcollections/binary_heap.rs

```diff
@@ -1200,6 +1236,15 @@ where T: Clone + Ord {
     place: vec::PlaceBack<'a, T>,
 }
 
+#[stable(feature = "collection_debug", since = "1.15.0")]
```

Comment:
`BinaryHeapPlace` is unstable so this impl should get the same stability attribute as `BinaryHeapPlace`.